File: 20210719/동전/동전.py
n, k = map(int, input().split())

coinList = []
for i in range(n):
    coinList.append(int(input()))

count = 0
coinList.reverse()

for i in coinList:
    if i <= k:
        count += k // i
        k = k % i
    if k == 0:
        print(count)
        break

File: 20210719/동전/동전.js
function solution(n, k, coinList) {
  let count = 0;
  coinList.reverse();

  for (const i of coinList) {
    if (i <= k) {
      count += parseInt(k / i);
      k = k % i;
    }
    if (k == 0) {
      console.log(cout);
      break;
    }
  }
}
